tox.ini

```diff
@@ -11,6 +11,7 @@ basepython =
 deps =
     cov: coverage>=4.3
     pytest
+    zopfli
```

Comment:
put zopfli in the `requirements.txt` and pin to the latest. Brotli is already in there. The requirements.txt is only used here to configure the test suite's environment in a reproducible way.

Reply:
unlike pytest or coverage which are test-only requirements, zopfli is an (optional) installation requirement, ie. required at runtime if one wants to export woff with it.

Reply:
OK, no problem. Will remove from tox.ini and add to requirements.txt